server/server/models/user.py

```diff
@@ -88,9 +88,17 @@ def get_role(self):
     @staticmethod
     def update_user(request, json_data):
         """ Method to update user in database """
-        if request.dbsession.query(User).get(request.matchdict['profile_id']):
+        check_dict = {
+            "first_name",
+            "last_name",
+            "nickname",
+            "location"
+        }
+        if request.dbsession.query(User).get(request.matchdict['profile_id']) \
+           and any(json_data.get(key) for key in check_dict):
```

Comment:
But why??? Everything was ok here, please don't touch this anymore and return back this stuff to view.